github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -257,7 +257,6 @@ presubmits:
       preset-shared-images: "true"
     max_concurrency: 2
     name: pull-kubevirt-e2e-k8s-1.22-sig-performance
-    optional: true
     run_if_changed: ^pkg/virt-api/.*|^pkg/virt-handler/.*|^pkg/virt-controller/.*|^pkg/virt-launcher/.*
```

Comment:
`run_if_changed` clashes with `always_run: true` - has to be removed in this case.

Reply:
Sorry - I wasn't clear enough - in this case I meant that `run_if_changed` should be removed.

Reply:
It would - i think this has come up previously - the branch protector in prow won't stop merges if conditionally run jobs fail so even with the latest `run_if_changed` it doesn't guarantee that it will stop the merge in a failed case. https://github.com/kubevirt/project-infra/pull/2249#issuecomment-1212089928